include/fastrtps/xmlparser/XMLEndpointParser.h

```diff
@@ -76,7 +77,7 @@ class StaticRTPSParticipantInfo
  * Class XMLEndpointParser used to parse the XML file that contains information about remote endpoints.
  * @ingroup DISCVOERYMODULE
  */
-class XMLEndpointParser
+class XMLEndpointParser : XMLParser
```

Comment:
This change is probably breaking ABI, so this PR should be considered for `2.6.0` instead of `2.5.1`.

Reply:
Yes,  you are right. I will change the associated milestone, but also I rise the question in a meeting.